modules/oe_multilingual_url_suffix/src/Plugin/LanguageNegotiation/LanguageNegotiationUrlSuffix.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation;
+
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\Core\Render\BubbleableMetadata;
+use Drupal\language\Plugin\LanguageNegotiation\LanguageNegotiationUrl;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class LanguageNegotiationUrlSuffix for identifying language via URL suffix.
+ *
+ * @LanguageNegotiation(
+ *   id = \Drupal\oe_multilingual_url_suffix\Plugin\LanguageNegotiation\LanguageNegotiationUrlSuffix::METHOD_ID,
+ *   types = {\Drupal\Core\Language\LanguageInterface::TYPE_INTERFACE,
+ *   \Drupal\Core\Language\LanguageInterface::TYPE_CONTENT,
+ *   \Drupal\Core\Language\LanguageInterface::TYPE_URL},
```

Comment:
I know that core does it this way but can't we indent these 3 lines? They'll be more readable:
```
 *   types = {
 *      \Drupal\Core\Language\LanguageInterface::TYPE_INTERFACE,
 *      \Drupal\Core\Language\LanguageInterface::TYPE_CONTENT,
 *      \Drupal\Core\Language\LanguageInterface::TYPE_URL
 *   },
```

Reply:
Done